### DISCUSSION TITLE
MAUI performance on Low End Devices

### DISCUSSION THREAD

**Discussion by dharamhbtik** [dotnet/maui#17617] [Q&A]
We are planning to migrate to MAUI from Xamarin Forms and most of the devices using the app are low end devices like Samsung X-Cover 4 or X-Cover 5.
I am wondering if there is any minimum hardware requirement for MAUI. 
I have below doubts : 

1. Will there be any performance improvement in the app compared to Xamarin forms ( on X Cover 4/5 devices)
2. How .Net 7/8 will  perform on low end devices?

Is it good to migrate to MAUI if we are with low end devices or move to some other frameworks like Flutter or React Native.

**Comment by mohachouch** [comment]
Same issue on Samsung Galaxy Tab2
